Instructions/Exercises/05-generate-images.md

```diff
@@ -7,30 +7,32 @@ lab:
 
 The Azure OpenAI Service includes an image-generation model named DALL-E. You can use this model to submit natural language prompts that describe a desired image, and the model will generate an original image based on the description you provide.
 
+In this exercise, you'll use a DALL-E version 3 model to generate images based on natural language prompts.
+
 This exercise will take approximately **25** minutes.
 
 ## Provision an Azure OpenAI resource
 
-Before you can use Azure OpenAI models, you must provision an Azure OpenAI resource in your Azure subscription.
+Before you can use Azure OpenAI to generate images, you must provision an Azure OpenAI resource in your Azure subscription. The resource must be in a region where DALL-E models are supported.
 
 1. Sign into the **Azure portal** at `https://portal.azure.com`.
 2. Create an **Azure OpenAI** resource with the following settings:
     - **Subscription**: *Select an Azure subscription that has been approved for access to the Azure OpenAI service, including DALL-E*
     - **Resource group**: *Choose or create a resource group*
-    - **Region**: **East US**\*
+    - **Region**: *Choose either **East US** or **Sweden Central***\*
     - **Name**: *A unique name of your choice*
     - **Pricing tier**: Standard S0
 
-    > \* DALL-E models are only available in Azure OpenAI service resources in the **East US** region.
+    > \* DALL-E 3 models are only available in Azure OpenAI service resources in the **East US** and **Sweden Central** regions.
 
 3. Wait for deployment to complete. Then go to the deployed Azure OpenAI resource in the Azure portal.
 
 ## Explore image-generation in the DALL-E playground
 
 You can use the DALL-E playground in **Azure OpenAI Studio** to experiment with image-generation.
 
-1. In the Azure portal, on the **Overview** page for your Azure OpenAI resource, use the **Explore** button to open Azure OpenAI Studio in a new browser tab. Alternatively, navigate to [Azure OpenAI Studio](https://oai.azure.com) directly.
-2. In the **Playground** section, select the **DALL-E** playground.
+1. In the Azure portal, on the **Overview** page for your Azure OpenAI resource, use the **Explore** button to open Azure OpenAI Studio in a new browser tab. Alternatively, navigate to [Azure OpenAI Studio](https://oai.azure.com) directly at `https://oai.azure.com`.
```

Comment:
suggestion: It seems redundant to have the same link twice in the same phrase. Also, this is not a Markdown link.

Reply:
The additional code-formatted link is used in our hosted lab environment to make it easier to auto-type the link into a hosted VM.

Reply:
Ah, I see, it's the *Click to type into the lab VM* feature 👍